APITests/utils.py

```diff
@@ -360,13 +365,11 @@ def get_access_token() -> str:
             str: a token to access asset store
         """
         # for running on github action
-        if "SYNAPSE_AUTH_TOKEN" in os.environ:
-            token = os.environ["SYNAPSE_AUTH_TOKEN"]
-            logger.debug("Successfully found synapse access token")
-        else:
-            token = os.environ["TOKEN"]
-        if token is None or "":
-            logger.error("Synapse access token is not found")
+        token = os.getenv("SYNAPSE_AUTH_TOKEN")
+        if token is None:
+            token = os.getenv("TOKEN")
```

Comment:
This can probably go away as well, but I'm just not sure how `TOKEN` is used else where